scripts/configure.js

```diff
@@ -82,12 +82,19 @@ console.log(`Installing llnode for ${lldbExe}, lldb version ${lldbVersion}`);
 // TODO: The llvm project is probably moving to github soon at that point we
 // should stop using the mirror.
 if (lldbHeadersBranch != undefined) {
-  console.log('Cloning lldb from ' + lldbHeadersBranch);
-  child_process.execSync(`rm -rf ${lldbIncludeDir}`);
-  child_process.execFileSync('git',
+  if (!fs.lstatSync(path.join(lldbIncludeDir, 'include')).isDirectory()) {    
+    console.log('Cloning lldb from ' + lldbHeadersBranch);  
+    child_process.execSync(`rm -rf ${lldbIncludeDir}`);
```

Comment:
I know it's not new but seeing `rm -rf` in an install script always gives me the creeps...

Reply:
Yea I thought about removing this...I guess we can just clone if the directory is not there, otherwise do nothing?

Reply:
Or use `rimraf`

Reply:
@indutny `script/configure.js` is run during preinstall so we can't require `rimraf` at that point..